config_file.py

```diff
@@ -118,6 +118,9 @@
 # If the overlap in the joint exceeds this value, warn the user [inches|mm]
 warn_overlap = {warn_overlap}
 
+# Cutting part of the bit %
```

Comment:
I don't understand "cutting part of the bit".  Why is that different than width?

Reply:
Let we need to make 1" grove with a 1/2" bit. We can make 2 passes but the center of grove would not be clean (that means gentle = 100%).  In case we made 3 cuts we got fine surface (we can made even more but it does not make much sense). So we can tell program to avoid full size cuts if possible. 1st cut is full size in any case. For the next cuts application will take "gentle into accout) and shift place next cut to value less then 1/2 (in case of gentle = 50% the shift will be 1/4") the 3rd (2st + 1/4") cut will finalize grove.

Reply:
Width means the width of grove not the width of a bit.
My English is not that good to delivery this tiny difference in comments.

Reply:
I understand, thank you.  And your English is a lot better than my Russian.